src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -118,8 +125,16 @@
   (when-not (string? (:path (:match-request rule)))
     (throw (IllegalArgumentException.
             (str "The path set in the authorization rule specified as "
-                 (pprint-rule (:path (:match-request rule))) " is invalid. "
+                 (pprint-rule rule) " is invalid. "
                  "It should be a string."))))
+  (when (and (not (nil? (:method rule)))
+             (nil? (some #(= % (:method rule)) valid-methods)))
+    (throw (IllegalArgumentException.
+             (str "The method specified in the authorization rule specified as "
+                  (pprint-rule rule) " is invalid. "
+                  "It should be either a string or list of strings that is equal "
+                  "to one of the following methods: 'get' 'post' 'put' 'delete' "
+                  "'head' 'any'"))))
```

Comment:
...at least that's the conclusion we reached during the Module meeting yesterday